README.md

```diff
@@ -1 +1,15 @@
-# Beyond-07-team-5
\ No newline at end of file
+# ResuMe
+
+# About
+ResuMe is an open-source platform for students and job seekers that allows users to share their resume for other people to give their intake on it and review it.
+# Getting started
+These instructions will get you a copy of the project up and running on your local machine.
+## Prerequires
+ - Install [Vagrant](https://www.vagrantup.com/downloads).
+ - Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads).
+## Initializing Virtual Environment 
+1. Clone/Download the [ResuMe](https://github.com/redhat-beyond/ResuMe) repository.
```

Comment:
You can add here the command for the clone if you wish to, something like:
Clone the [ResuMe](https://github.com/redhat-beyond/ResuMe) repository using the command
```
git clone https://github.com/redhat-beyond/ResuMe.git
```